base_app/src/tools/logger.ts

```diff
@@ -139,12 +136,12 @@ export class ClientLogger {
 
 export enum LogLevel {
     EMERGENCY,
-    ALERT,
+//    ALERT,
     CRITICAL,
-    ERROR,
-    WARNING,
+//    ERROR,
```

Comment:
If these are no longer needed remove instead of comment out. But wouldn't an error level still be needed?

Reply:
I think we're not currently using ERROR in the typescript logger.  It's commented because it seems likely to be used at some point, but don't want it to show "unused" in the IDE.